.rancher-pipeline.yml

```diff
@@ -0,0 +1,31 @@
+stages:
```

Comment:
Please make sure this removed before merge to master.

Reply:
yes sure